apps/base-docs/docs/tools/cross-chain.md

```diff
@@ -44,6 +47,16 @@ To view current transactions and live stats about the Axelar network please visi
 
 ---
 
+## Crossmint
+[Crossmint](https://crossmint.com/?utm_source=backlinks&utm_medium=docs&utm_campaign=base) allows you to create and deploy NFT Collections and enable cross-chain payments. This enables your users and customers to purchase an NFT from a collection deployed on Base using Ethereum or Solana tokens. 
+
+Check out [Crossmint Docs](https://docs.crossmint.com/docs/cross-chain-payments/?utm_source=backlinks&utm_medium=docs&utm_campaign=base) to learn more about how you can integrate Crossmint with your product to power cross-chain payments. 
+
+#### Supported Networks
+- [Base Mainnet](https://www.crossmint.com/products/nft-checkout/?utm_source=backlinks&utm_medium=docs&utm_campaign=base)
+- [Base Goerli](https://www.crossmint.com/products/nft-checkout/?utm_source=backlinks&utm_medium=docs&utm_campaign=base) (Testnet)
```

Comment:
@rohit-710

Heya! We are deprecating support for Base Goerli in the near future; do you all support Base Sepolia and if so, could you please update this link?

Reply:
Hey @wbnns 
We do not support Base Sepolia at the moment, but I can update it when we release support for it!

Reply:
@rohit-710 Ok! Let's go ahead and drop the Base Goerli reference for now, then, since the network will be deprecated in the next couple of weeks[1].

Could you please update?

[1] https://base.mirror.xyz/-1DzslYOS7HRrbOrHdcz5sZmfIF_tpxJ-NRvcMmZ75c

Reply:
Hey @wbnns, I just updated it! It should be good to go now. 

Let me know if you require me to make any other changes